### ISSUE TITLE
By opening the given file, you should see the property chapter=true on top 

### ISSUE THREAD

**Issue by sebhewelt** [matcornic/hugo-theme-learn#66]
`hugo new --kind chapter basics/_index.md`

> By opening the given file, you should see the property chapter=true on top, meaning this page is a chapter.

Although there is `chapter.md` file in `themes/hugo-theme-learn/archetypes`, it is not applied to be the archetype for `new --kind chapter`. The default.md from ~/archetypes is applied instead.

The tutorial doesn't say it - should i copy the folders from `themes/hugo-theme-learn` to the root folder?

I thought the hugo engine would look into the themes/hugo-theme-learn/archetypes after it checks ~/archetypes, but it doesn't

**Comment by matcornic** [comment]
Hello,

Can you explain what are the commands and steps you execute ?
I just tested it and it works well as expected : http://recordit.co/RY5lqSCOG8

**Comment by sebhewelt** [comment]
Oh ok, big thanks, the tutorial doesn't say I should cd to `exampleSite ` directory. All clear now :)

**Comment by matcornic** [comment]
`exampleSite` is the documentation of the theme. In the tutorial, your are supposed to create your own site (then you don't need to `cd` to exampleSite because your are already in it) : https://learn.netlify.com/en/basics/installation/#create-your-project

**Comment by sebhewelt** [comment]
Yeah, i know, and i made my own site. But that's why I opened the issue in the first place, it didn't work on my hugo site.

http://recordit.co/wiY8iJBTl7

Maybe I skipped a step somewhere, but the line:

> By opening the given file, you should see the property chapter=true on top, meaning this page is a chapter.

doesn't apply to my site. Could you take a look at the vid?

**Comment by khalidabuhakmeh** [comment]
I figured it out. @sebhewelt is right that running `hugo new --kind chapter basics/_index.md` doesn't work, but that's because the `default.md` file is taking precedence over any kind you specify (this could be a bug) in hugo itself.

Delete the `default.md` under your sites `archetypes` folder and it should work now.

**Comment by khalidabuhakmeh** [comment]
https://discourse.gohugo.io/t/hugo-doesnt-use-theme-archetypes/8382/20
